src/main/java/com/j256/cloudwatchlogbackappender/CloudWatchAppender.java

```diff
@@ -505,29 +529,6 @@ public void run() {
 					events.clear();
 				}
 			}
-
-			/*
-			 * We have been interrupted so write all of the rest of the events and then quit
-			 */
-
-			events.clear();
```

Comment:
We shouldn't clear `events` because it can contain some messages which were not pushed into CloudWatch yet.